buildSrc/src/main/kotlin/land/tbp/wwsc/tralala.gradle.kts

```diff
@@ -1,82 +0,0 @@
-package land.tbp.wwsc
-import land.tbp.wwsc.Versions.springBootVersion
-import land.tbp.wwsc.Versions.h2Version
-import land.tbp.wwsc.Versions.openApiVersion
-import land.tbp.wwsc.Versions.springMockkVersion
-import land.tbp.wwsc.Versions.assertJVersion
-import land.tbp.wwsc.Versions.awaitilityVersion
-
-plugins {
-    kotlin("jvm")
-    id("org.springframework.boot")
```

Comment:
hey @guai these plugins are not applied afaics. Could you update the MR to show me how to add all of these?

Reply:
These plugins should be applied to all subprojects which apply the "tralala" plugin (which you renamed to "MyVersionsPlugin")